Makefile

```diff
@@ -368,6 +368,7 @@ generate-manifests-core: $(CONTROLLER_GEN) $(KUSTOMIZE)
 		paths=./webhooks/... \
 		paths=./$(EXP_DIR)/api/... \
 		paths=./$(EXP_DIR)/controllers/... \
+		paths=./$(EXP_DIR)/internal/... \
```

Comment:
I think we need a follow-up issue to cleanup the gen targets. We moved a few more controllers and for some reason `generate-manifests-cabpk` doesn't generate RBAC, which is why we didn't saw that we're using the wrong (aka old) path there.

Reply:
> I think we need an additional follow-up issue to cleanup the gen targets. We moved a few more controllers where we didn't use those paths/ delete the old one. Additionally, e.g. `generate-manifests-cabpk` doesn't generate RBAC anymore because we're not using any paths with rbac annotations anymore (also doesn't lead to verify errors that way)

Created a PR to fix the existing targets: https://github.com/kubernetes-sigs/cluster-api/pull/5696

Reply:
@shivanshu1333 Can you please update l.370 to `paths=./$(EXP_DIR)/internal/controllers/... \`

Reply:
done!